airbyte-integrations/connectors/source-klaviyo/source_klaviyo/manifest.yaml

```diff
@@ -0,0 +1,336 @@
+version: 0.47.2
+type: DeclarativeSource
+definitions:
+  schema_loader:
+    type: JsonFileSchemaLoader
+    file_path: "./source_klaviyo/schemas/{{ parameters.name }}.json"
+
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: [data]
+
+  legacy_api_requester:
+    type: HttpRequester
+    url_base: https://a.klaviyo.com/api/v1/
+    http_method: GET
+    request_headers: {}
+    authenticator:
+      type: ApiKeyAuthenticator
+      api_token: '{{ config[''api_key''] }}'
+      inject_into:
+        type: RequestOption
+        inject_into: request_parameter
+        field_name: api_key
+    request_body_json: {}
+
+  revised_api_requester:
+    type: HttpRequester
+    url_base: https://a.klaviyo.com/api/
+    http_method: GET
+    request_headers:
+      revision: "{{ parameters.revision }}"
+      Accept: application/json
+      Content-Type: application/json
+    authenticator:
+      type: ApiKeyAuthenticator
+      api_token: 'Klaviyo-API-Key {{ config[''api_key''] }}'
+      inject_into:
+        type: RequestOption
+        inject_into: header
+        field_name: Authorization
+    request_body_json: {}
+
+  base_retriever:
+    type: SimpleRetriever
+    record_selector:
+      $ref: "#/definitions/selector"
+
+  base_stream:
+    type: DeclarativeStream
+    schema_loader:
+      $ref: "#/definitions/schema_loader"
+
+  page_increment_paginator:
+    type: DefaultPaginator
+    page_size_option:
+      type: RequestOption
+      field_name: count
+      inject_into: request_parameter
+    page_token_option:
+      type: RequestOption
+      inject_into: request_parameter
+      field_name: page
+    pagination_strategy:
+      type: PageIncrement
+      start_from_page: 0
+      page_size: 100
+
+  cursor_paginator:
+    type: DefaultPaginator
+    page_token_option:
+      type: RequestPath
+    page_size_option:
+      inject_into: request_parameter
+      type: RequestOption
+      field_name: page[size]
+    pagination_strategy:
+      type: CursorPagination
+      page_size: 100
+      cursor_value: '{{ response.get("links", {}).get("next", {}) }}'
+      stop_condition: '{{ not response.get("links", {}).get("next", {}) }}'
+  
+  base_incremental_sync:
+    type: DatetimeBasedCursor
+    cursor_field: "{{ parameters.cursor_field }}"
+    cursor_datetime_formats:
+      - "{{ parameters.cursor_datetime_formats }}"
```

Comment:
```
[35m44:[0m [4.56s] {"type": "TRACE", "trace": {"type": "ERROR", "emitted_at": 1693504634501.5, "error": {"message": "Something went wrong in the connector. See the logs for more details.", "internal_message": "No format in ['{{ parameters.cursor_datetime_formats }}'] matching 2021-05-18 01:29:51", "stack_trace": "Traceback (most recent call last):\n  File \"/airbyte/integration_code/main.py\", line 13, in \n    launch(source, sys.argv[1:])\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py\", line 200, in launch\n    for message in source_entrypoint.run(parsed_args):\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py\", line 114, in run\n    yield from map(AirbyteEntrypoint.airbyte_message_to_string, self.read(source_spec, config, config_catalog, state))\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/entrypoint.py\", line 156, in read\n    yield from self.source.read(self.logger, config, catalog, state)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/declarative/manifest_declarative_source.py\", line 136, in read\n    yield from super().read(logger, config, catalog, state)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py\", line 142, in read\n    raise e\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py\", line 122, in read\n    yield from self._read_stream(\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py\", line 197, in _read_stream\n    for record in record_iterator:\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/abstract_source.py\", line 328, in _read_full_refresh\n    for record_data_or_message in record_data_or_messages:\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/declarative/declarative_stream.py\", line 104, in read_records\n    yield from self.retriever.read_records(stream_slice)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/declarative/retrievers/simple_retriever.py\", line 305, in read_records\n    most_recent_record_from_slice = self._get_most_recent_record(most_recent_record_from_slice, stream_data, stream_slice)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/declarative/retrievers/simple_retriever.py\", line 319, in _get_most_recent_record\n    return current_most_recent if self.cursor.is_greater_than_or_equal(current_most_recent, record) else record\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/declarative/incremental/datetime_based_cursor.py\", line 283, in is_greater_than_or_equal\n    return self.parse_date(first_cursor_value) >= self.parse_date(second_cursor_value)\n  File \"/usr/local/lib/python3.9/site-packages/airbyte_cdk/sources/declarative/incremental/datetime_based_cursor.py\", line 196, in parse_date\n    raise ValueError(f\"No format in {self.cursor_datetime_formats} matching {date}\")\nValueError: No format in ['{{ parameters.cursor_datetime_formats }}'] matching 2021-05-18 01:29:51\n", "failure_type": "system_error"}}}
```

Reply:
What stream is this error for? @marcosmarxm

Reply:
Got it